src/desktop/apps/consign/client/analytics_middleware.ts

```diff
@@ -1,24 +1,13 @@
 import * as actions from "./actions"
 import { data as sd } from "sharify"
 
-const _analyticsHooks = require("../../../lib/analytics_hooks.coffee")
-
-// FIXME: Rewire
-let analyticsHooks = _analyticsHooks
-
 const analyticsMiddleware = store => next => action => {
   const result = next(action)
   const nextState = store.getState()
 
   // track certain types of actions
   switch (action.type) {
     case actions.UPDATE_USER: {
-      const { user, accountCreated } = action.payload
-      analyticsHooks.trigger("consignment:account:created", {
-        id: user.id,
-        email: user.email,
-        accountCreated: accountCreated,
-      })
```

Comment:
We're no longer tracking consignment account creation?

Reply:
This is a duplicate. Consign used to have a custom auth implementation, and this event was not removed when we updated the flow to use the global auth component.

Reply:
I see, so it's tracked in a separate component 👍